### ISSUE TITLE
link in markdown is not correctly resolved when using relative paths

### ISSUE THREAD

**Issue by BigMichi1** [bsorrentino/maven-confluence-plugin#212]
i have the following directory structure
    
![Screenshot_20200314_100914](https://user-images.githubusercontent.com/91786/76678872-e3a97c00-65db-11ea-9391-315c43fbcbb8.png)

the `site.xml` descriptor in the confluence directory has this content

```xml
<?xml version="1.0" encoding="UTF-8"?>
<bsc:site
        xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
        xmlns:bsc='https://github.com/bsorrentino/maven-confluence-plugin'
        xsi:schemaLocation='https://github.com/bsorrentino/maven-confluence-plugin https://raw.githubusercontent.com/bsorrentino/maven-confluence-plugin/master/schemas/site-schema-6.0.xsd'>
    <home uri="index.md">
        <child name="Checks" uri="../markdown/checks.md">
            <child name="Disallowed Annotations" uri="../markdown/check_disallowed_annotations.md"/>
        </child>
    </home>
</bsc:site>
```

the `checks.md` file
```markdown
# Checks

- [Disallowed Annotations](check_disallowed_annotations.md)
```

and the `check_disallowed_annotations.md`
```markdown
# Disallowed
```

the problem now is when the pages are deployed to confluence the link in the `checks.md` is not correctly resolved, instead i have in confluence a link that creates a new page named "check_disallowed_annotations.md" when clicking on it.

when i move both files to the confluence directory and deploy it, then the links are correct. so it seems that the links are not working correctly when the files are not in the same directory where the `site.xml` is located. is there something i need to change in the `site.xml` or in the plugin configuration?

**Comment by bsorrentino** [comment]
Hi @BigMichi1 

Which version of plugin are you using ?

**Comment by BigMichi1** [comment]
its version 6.7.2

also i figured out that the 
<childrenTitlesPrefixed>false</childrenTitlesPrefixed>
does not create the correct links in markdown files, but maybe that's a different issue because the prefix is applied every time to child pages

**Comment by bsorrentino** [comment]
hi @BigMichi1 

I'll verify soon

**Comment by bsorrentino** [comment]
hi @BigMichi1, sorry for late in answer.

to fix your issue it is enough to specify `name` on home page and in the link you have to avoid to specify the source page but just the title. Below an example

the `site.xml` file
```xml
<?xml version="1.0" encoding="UTF-8"?>
<bsc:site
        xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
        xmlns:bsc='https://github.com/bsorrentino/maven-confluence-plugin'
        xsi:schemaLocation='https://github.com/bsorrentino/maven-confluence-plugin https://raw.githubusercontent.com/bsorrentino/maven-confluence-plugin/master/schemas/site-schema-6.0.xsd'>
    <home name="index" uri="index.md">
        <child name="Checks" uri="../markdown/checks.md">
            <child name="Disallowed Annotations" uri="../markdown/check_disallowed_annotations.md"/>
        </child>
    </home>
</bsc:site>
```

the `check.md` file
```markdown
# Checks

- [Disallowed Annotations]
```

the `check_disallowed_annotations.md` file
```markdown
# Disallowed
```

Concerning `<childrenTitlesPrefixed>false</childrenTitlesPrefixed>` I've verified that seems that it is ignored on links processing (I've opened issue #215 for that)

**Comment by bsorrentino** [comment]
Hi @BigMichi1,  I've investigate deeply and this is really  a bug because it should be work as you have underlined and not like I've suggested above.

I'm working on... to solve the problem

**Comment by bsorrentino** [comment]
fixed in release `6.8`

**Comment by BigMichi1** [comment]
good thing, now it works, thank you
